src/main/webapp/mentors.js

```diff
@@ -122,3 +122,32 @@ function createProjectsDiv(projectIDs) {
   }
   return projectsDiv;
 }
+
+function submitForm() {
+  const inputUrl = document.getElementById("inputRepo").value;
+  const encodedBody = new URLSearchParams();
+  encodedBody.append('inputRepo', inputUrl);
+  let url = new URL(
+      '/mentors', window.location.protocol + '//' + window.location.hostname);
```

Comment:
nit: Use [window location origin](https://developer.mozilla.org/en-US/docs/Web/API/Location/origin) here.
```suggestion
  let url = new URL('/mentors', window.location.origin);
```